quantum-adapters/kiwoom-adapter/src/kiwoom_api/analysis/core/area_scorers.py

```
@@ -0,0 +1,444 @@
+try:
+    from .vlookup_calculator import VLOOKUPCalculator
+    from .evaluation_criteria import EvaluationCriteria
+except ImportError:
+    # If relative imports fail, add src to path and use absolute imports
+    src_path = Path(__file__).parent.parent.parent
+    if str(src_path) not in sys.path:
+        sys.path.insert(0, str(src_path))
+    from kiwoom_api.analysis.core.vlookup_calculator import VLOOKUPCalculator
+    from kiwoom_api.analysis.core.evaluation_criteria import EvaluationCriteria
+
+from typing import Dict, Any, List
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)